### ISSUE TITLE
Links are relative by default for 1.3.10

### ISSUE THREAD

**Issue by StirStudios** [harlan-zw/nuxt-seo-kit#93]
### Describe the bug

The homepage for example produces a canonical link of `/` when it should be using absolute urls? This progresses when you generate other pages such as `/about`

In order to resolve this we added the following:

```
const route = useRoute()

useHead({
  link: [
    {
      rel: 'canonical',
      href: useRuntimeConfig().public.siteUrl + route.fullPath,
    },
  ],
})
```

to the template.

Perhaps I configured something incorrectly or maybe relative is the chosen route.

### System / Nuxt Info

```Shell
"@nuxt/devtools": "^0.8.2"
"@nuxt/image": "rc"
"nuxt": "^3.7.0"
"nuxt-seo-kit": "^1.3.10"
```

Also noticed this is happening on og:url as well.

**Comment by StirStudios** [comment]
More info to help.

This has only happened since upgrading to `nuxt-seo-kit 1.3.10`. As soon as we downgrade to `nuxt-seo-kit 1.3.9` the problem is resolved.

**Comment by harlan-zw** [comment]
Thanks, this should be fixed in 1.3.11
